src/vochain/vochain.proto

```diff
@@ -207,6 +221,58 @@ message RegisterKeyTx {
         string weight = 5;   // Weight to delegate to newKey
 }
 
+message MintTokensTx {
+	TxType txtype = 1;
+	bytes nonce = 2;
+	bytes to = 3;
+	bytes value = 4;
+}
+
+message SendTokensTx {
+	TxType txtype = 1;
+	bytes nonce = 2;
```

Comment:
How do we know if a nonce should be `bytes` or `uint32` ? Is because the `bytes` represents the random nonce value and the `uint32` represents the incremental nonce value ?

Reply:
Yes. If the motivation for the nonce is to add entropy, then bytes. If it is a sequence number, then uint32